docs/guides/dpdk-guide.md

```diff
@@ -69,16 +71,16 @@ appropriate path instead of `./install`.
 copying files to system directories.
 
 ```bash
-source ./scripts/dpdk/setup_env.sh $IPDK_RECIPE $SDE_INSTALL $DEPEND_INSTALL
-sudo ./scripts/dpdk/copy_config_files.sh $IPDK_RECIPE $SDE_INSTALL
+source $P4CP_INSTALL/sbin/setup_env.sh $P4CP_INSTALL $SDE_INSTALL $DEPEND_INSTALL
+sudo bash $P4CP_INSTALL/sbin/copy_config_files.sh $P4CP_INSTALL $SDE_INSTALL
 ```
 
 #### Set hugepages required for DPDK
 
 Run the hugepages script.
 
 ```bash
-sudo ./scripts/dpdk/set_hugepages.sh
+sudo bash $P4CP_INSTALL/sbin/set_hugepages.sh
```

Comment:
Why are you injecting a `bash` command? If set_hugepages.sh has execute permission, you should be able to execute it directly.

This question applies throughout this document.

Reply:
I was seeing an error when I didn't have bash. But now when I try this without bash, I don't see the error. Let me update to remove bash

Reply:
Removed

Reply:
> I was seeing an error when I didn't have bash. But now when I try this without bash, 

You might have had a script that didn't have execute permission.